GDJS/Runtime/runtimescene.ts

```diff
@@ -748,6 +764,20 @@ namespace gdjs {
       if (sceneName) this._requestedScene = sceneName;
     }
 
+    requestLoad(
+      request: boolean,
+      sceneVar?: gdjs.Variable,
+      storageName?: string
+    ): void {
+      this._isLoadingRequested = request;
+      if (storageName) {
+        this._loadStorageName = storageName;
+      }
+      if (sceneVar) {
+        this._loadVariable = sceneVar;
```

Comment:
are those values cleared the next time we use the function without the `sceneVar` and `storageName`?

If not, I think we should always set the value from what's coming from the props of the function.
This will allow setting `undefined` or whatever string/variable is passed